core-bundle/contao/templates/modules/mod_navigation.html5

```diff
@@ -1,6 +1,6 @@
 
 <!-- indexer::stop -->
-<nav class="<?= $this->class ?> block"<?= $this->cssID ?><?php if ($this->style): ?> style="<?= $this->style ?>"<?php endif; ?>>
+<nav class="<?= $this->class ?> block"<?= $this->cssID ?><?php if ($this->style): ?> style="<?= $this->style ?>"<?php endif; ?><?php if ($this->ariaLabel): ?> aria-label="<?= $this->ariaLabel ?>"<?php endif; ?>>
```

Comment:
```suggestion
<nav<?= $this->cssID ?><?= $this->attr()->addClass($this->class)->setIfExists('style', $this->style)->setIfExists('aria-label', $this->ariaLabel) ?>>
```

@leofeyer wdyt? Should we do it like that for all templates, now that we have `$this->attr()`?

Reply:
Yes, why not. Do we need a `->setIfExists('id', $this->???)` instead of `$this->cssID` as well then?

Reply:
Well, the `$this->???` is the problem :). The CSS ID is currently not available in the template. Only the string including the attribute (i.e. `' id="foobar"'`). We would introduce a `rawCssID` variable or something and pass `$cssID[0]` to that variable in various places, so that we can use `->setIfExists('id', $this->rawCssID)`.

Another option would be to use:

```php
$this->attr($this->cssID)
    ->addClass($this->class)
    ->addStyle($this->style)
    ->setIfExists('aria-label', $this->ariaLabel)
```

This should automatically parse the `id="..."` string and add it as an attribute (if applicable).

Reply:
That would be nice if it works.

Reply:
Just tested it - it does work.

Reply:
`$this->attr()…->mergeWith($this->cssID)` would also work, JFI ☺️

Reply:
I'd prefer passing it to `attr()`.

Reply:
See https://github.com/contao/contao/pull/7218